.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "pac-cmd"]
```

Comment:
I really dislike submodules.  We used a submodule for lantern-ui on the old Java Lantern and it was a lot of trouble.  Can you just put pac and pac-cmd both in flashlight-build?